internal/commands/loglevel/set_test.go

```diff
@@ -17,46 +17,55 @@ limitations under the License.
 package loglevel_test
 
 import (
-	"context"
+    "context"
 
-	. "github.com/onsi/ginkgo/v2"
-	. "github.com/onsi/gomega"
-	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+    . "github.com/onsi/ginkgo/v2"
+    . "github.com/onsi/gomega"
+    metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+    "sigs.k8s.io/controller-runtime/pkg/client/fake"
 
-	libsveltosv1alpha1 "github.com/projectsveltos/libsveltos/api/v1alpha1"
-	"github.com/projectsveltos/sveltosctl/internal/commands/loglevel"
-	"github.com/projectsveltos/sveltosctl/internal/utils"
+    libsveltosv1alpha1 "github.com/projectsveltos/libsveltos/api/v1alpha1"
+    "github.com/projectsveltos/sveltosctl/internal/commands/loglevel"
+    "github.com/projectsveltos/sveltosctl/internal/utils"
 )
 
 var _ = Describe("Set", func() {
-	It("set updates default DebuggingConfiguration instance", func() {
-
-		scheme, err := utils.GetScheme()
-		Expect(err).To(BeNil())
-		c := fake.NewClientBuilder().WithScheme(scheme).Build()
-
-		utils.InitalizeManagementClusterAcces(scheme, nil, nil, c)
-		Expect(loglevel.UpdateDebuggingConfiguration(context.TODO(), libsveltosv1alpha1.LogLevelDebug,
-			string(libsveltosv1alpha1.ComponentAddonManager))).To(Succeed())
-
-		k8sAccess := utils.GetAccessInstance()
-
-		currentDC, err := k8sAccess.GetDebuggingConfiguration(context.TODO())
-		Expect(err).To(BeNil())
-		Expect(currentDC).ToNot(BeNil())
-		Expect(currentDC.Spec.Configuration).ToNot(BeNil())
-		Expect(len(currentDC.Spec.Configuration)).To(Equal(1))
-		Expect(currentDC.Spec.Configuration[0].Component).To(Equal(libsveltosv1alpha1.ComponentAddonManager))
-		Expect(currentDC.Spec.Configuration[0].LogLevel).To(Equal(libsveltosv1alpha1.LogLevelDebug))
-
-		Expect(loglevel.UpdateDebuggingConfiguration(context.TODO(), libsveltosv1alpha1.LogLevelInfo,
-			string(libsveltosv1alpha1.ComponentAddonManager))).To(Succeed())
-		currentDC, err = k8sAccess.GetDebuggingConfiguration(context.TODO())
-		Expect(err).To(BeNil())
-		Expect(currentDC).ToNot(BeNil())
-		Expect(currentDC.Spec.Configuration).ToNot(BeNil())
-		Expect(len(currentDC.Spec.Configuration)).To(Equal(1))
-		Expect(currentDC.Spec.Configuration[0].Component).To(Equal(libsveltosv1alpha1.ComponentAddonManager))
-		Expect(currentDC.Spec.Configuration[0].LogLevel).To(Equal(libsveltosv1alpha1.LogLevelInfo))
-	})
+    It("set updates default DebuggingConfiguration instance", func() {
+        scheme, err := utils.GetScheme()
+        Expect(err).To(BeNil())
+        c := fake.NewClientBuilder().WithScheme(scheme).Build()
+
+        utils.InitalizeManagementClusterAcces(scheme, nil, nil, c)
+        
+        dc := &libsveltosv1alpha1.DebuggingConfiguration{
+            ObjectMeta: metav1.ObjectMeta{
+                Name: utils.DefaultInstanceName,
+            },
+        }
+        err = c.Create(context.TODO(), dc)
+        Expect(err).To(BeNil())
+
+        Expect(loglevel.UpdateDebuggingConfiguration(context.TODO(), libsveltosv1alpha1.LogLevelDebug,
```

Comment:
the only change you need to make to those tests is to add extra argument as empty